changelogs/fragments/8428-assign-auth-flow-by-name-keycloak-client.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

Since this is more a feature than a bugfix, can you please adjust the changelog category:
```suggestion
minor_changes:
```